src/coreclr/jit/importer.cpp

```diff
@@ -5190,6 +5300,13 @@ NamedIntrinsic Compiler::lookupNamedIntrinsic(CORINFO_METHOD_HANDLE method)
         result = SimdAsHWIntrinsicInfo::lookupId(&sig, className, methodName, enclosingClassName, sizeOfVectorT);
     }
 #endif // FEATURE_HW_INTRINSICS
+    else if ((strcmp(namespaceName, "System.Runtime.CompilerServices") == 0) && (strcmp(className, "RuntimeHelpers") == 0))
+    {
+        if (strcmp(methodName, "CreateSpan") == 0)
+        {
+            result = NI_System_Runtime_CompilerServices_RuntimeHelpers_CreateSpan;
```

Comment:
Can we make it must-expand intrinsic for NativeAOT? (It is what we do for InitializeArray.)

Reply:
Yeah, that should be safe How do we identify NativeAOT vs ReadyToRun in the JIT. Do we do something like `opts.jitFlags->IsSet(JitFlags::JIT_FLAG_PREJIT) && !opts.jitFlags->IsSet(JitFlags::JIT_FLAG_READYTORUN)`? Unlike the InitializeArray intrinsic, CreateSpan intrinsic is entirely JIT handled, so the mechanism we used for InitializeArray doesn't work here.

Reply:
`IsTargetAbi(CORINFO_CORERT_ABI)` may be the most straightforward way to check for it.